src/constants/siteData.js

```diff
@@ -0,0 +1,8 @@
+const siteName = 'health-e';
+const siteDesc1 =
+	'Connect with friends around the world with great health topics';
+const user = 'User';
+const medical = 'Medical';
+const signup = 'Sign up';
+
+export { siteName, siteDesc1, user, medical, signup };
```

Comment:
there should not be a `constants` folder. Each component has its own `constants.js` file into its folder hierarchy. Reference the [Enye Conventions](https://github.com/enyeInc/Engineering-Conventions#file-structure) to see how you should do this